src/panels/invoke.html

```diff
@@ -159,6 +159,14 @@ <h4>Invoke contract method: '<span class="methodName"></span>'</h4>
                 <td width="20%">As data:</td>
                 <th class="asData"></th>
             </tr>
+            <tr>
+                <td width="20%">As address:</td>
```

Comment:
For future consideration, we should the formats for rendering contract outputs matches between the debugger and devtracker. I don't think debugger has a way of rendering the output as an address